frontend/packages/patternfly/src/components/notification-drawer/notification-entry.tsx

```diff
@@ -19,6 +19,21 @@ export enum NotificationTypes {
   update = 'update',
 }
 
+const notificationTypeString = (type: NotificationTypes): string => {
```

Comment:
You should declare this method inside the component since it's only used in one place. That way you can use the `t` from `useTranslation` instead of the global `i18next` instance.

Reply:
sounds good @spadgett